airbyte-cdk/python/airbyte_cdk/sources/declarative/requesters/error_handlers/http_response_filter.py

```diff
@@ -33,6 +33,7 @@ class HttpResponseFilter:
     config: Config
     parameters: InitVar[Mapping[str, Any]]
     action: Optional[Union[ResponseAction, str]] = None
+    failure_type: Optional[Union[FailureType, str]] = None
```

Comment:
Can you add it to the class documentation above in the attributes section?

Reply:
added